spec/appCacheHandler.spec.ts

```diff
@@ -0,0 +1,36 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { cleanUpAppCache, createAppCacheFile } from '../src/app/app-cache-handler';
+import { app, session } from './__mocks__/electron';
+
+jest.mock('fs', () => ({
+        writeFileSync: jest.fn(),
+        existsSync: jest.fn(() => true),
+        unlinkSync: jest.fn(),
+}));
+
+describe('app cache handler', () => {
+    const cachePathExpected = path.join(app.getPath('userData'), 'CacheCheck');
+
+    it('should call `cleanUpAppCache` correctly', () => {
+        const spyFn = 'unlinkSync';
+        const spy = jest.spyOn(fs, spyFn);
+        cleanUpAppCache();
+        expect(spy).toBeCalledWith(cachePathExpected);
+    });
+
+    it('should call `clearCache` when `session.defaultSession` is different from null', () => {
```

Comment:
please change this statement to `should call clearCache when session.defaultSession is not null`